file-system-lesson-11/main.js

```diff
@@ -0,0 +1,87 @@
+const fs = require('fs');
+
+const PATH = './example.txt';
+
+const readStream = fs.createReadStream(PATH);
+
+function readFileSync() {
+  try {
+    const content = fs.readFileSync(PATH, 'utf8');
```

Comment:
methods with "sync" will block thread. better use async version, play around with alternative methods with callbacks and promises